platforms-serverless/lambda-node-20/run.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/sh
+
+set -eux
```

Comment:
With a small adjustment, this should ensure that we return a non-zero exit code if the script breaks halfway:

```suggestion
set -euox pipefail
```